### DISCUSSION TITLE
Is it possible to specify different options for different containers?

### DISCUSSION THREAD

**Discussion by euberdeveloper** [containrrr/watchtower#1190] [Q&A] [answered]
I am using watchtower for different containers. The point is this, some containers, such as databases, should (absolutely) not have their volumes erased after their update and restart. Other containers, instead, would need their volumes to be erased.

For now I have seen that the available option is just an all-or-nothing thing. Maybe could I use the "Filter by scope" option with two different values "clean-me" and "keep-me"  and need to start two different instances of watchtower, one for the keep-volumes and one for the clean one? I deem this as an awkward solution, though.

**Comment by simonaronsson** [comment, marked as answer]
Yeah, that is the intended use of filtering by scope. 👍🏽
